fetch/api/abort/general.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>General fetch abort tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/common/utils.js"></script>
+  <script src="general.js"></script>
+</head>
+<body>
+<script>
+  fetch_tests_from_worker(new Worker("general.js"));
+  if (window.SharedWorker) {
+    fetch_tests_from_worker(new SharedWorker("general.js"));
+  }
```

Comment:
FWIW I found generating multiple tests with a wrapper simpler than contextualTestName: see https://github.com/w3c/web-platform-tests/tree/master/streams readme and https://github.com/w3c/web-platform-tests/blob/master/streams/generate-test-wrappers.js